ui/app/src/views/ViewDashboardList.tsx

```diff
@@ -57,7 +57,7 @@ function RenderDashboardList() {
     return <CircularProgress />;
   }
 
-  if (data === undefined) return null;
+  if (data === undefined || data === null) return null;
```

Comment:
Ah is this what was breaking your local dev environment?

Reply:
yeah :). But it was because the API didn't return an empty array, but returned `null` instead. It's fixed now.
I kept these modifications for safety.